nautobot/core/views/generic.py

```diff
@@ -101,6 +103,19 @@ def get(self, request, *args, **kwargs):
         Generic GET handler for accessing an object.
         """
         instance = get_object_or_404(self.queryset, **kwargs)
+        # Get the ObjectChange records to populate the advanced tab information
+        object_change_records = get_changes_for_model(instance)
+        created_by = None
+        last_updated_by = None
+        try:
+            created_by_record = object_change_records.get(action=ObjectChangeActionChoices.ACTION_CREATE)
+            created_by = created_by_record.user
```

Comment:
It might be fine to use `.user_name` instead of `.user` (the former is a string, and will persist even if the user account itself is later deleted)?